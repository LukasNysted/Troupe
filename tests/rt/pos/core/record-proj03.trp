let val r = {a = 5, b = 42, c = 10}
in r.z
end