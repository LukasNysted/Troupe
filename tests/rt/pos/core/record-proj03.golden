2024-02-14T11:34:02.654Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread b84af998-b70c-4bd2-b9e2-be2e4e56529e@{}%{}
>> record {a=5@{}%{}, b=42@{}%{}, c=10@{}%{}} does not have field 'z'
